array-manipulation-1/src/reverse.js

```diff
@@ -0,0 +1,12 @@
+/* exported reverse */
+function reverse(array) {
+  var reverseArray = [];
+  for (var i = array.length - 1; i >= 0; i--) {
+    if (array.length === 0) {
+      break;
+    } else {
+      reverseArray.push(array[i]);
+    }
```

Comment:
You can remove this conditional logic, since your loop condition expression is `i >= 0`. In the case that the array is empty (length of `0`), then `i` will be `-1`, which is not `>= 0`.